### ISSUE TITLE
What to do, if there's no hooks?

### ISSUE THREAD

**Issue by churroloca** [HIllya51/LunaHook#25]
### Question 问题

Hello,
We'd just talked a few moments ago. I hope that I'm doing this correctly.

I got plugins to work correctly but I still cannot find a hook, for a game. Does this mean the game doesn't have any hooks? This is the first game I've encountered this problem with. I'm still learning these types of programs.

I hope I don't sound rude, demanding or anything of the sort.

Thank you for your time,

**Comment by HIllya51** [comment]
Create the issue follow this template.
And you should directly give me the game resource rather than let me to find it by myself.


![屏幕截图 2024-04-27 221210](https://github.com/HIllya51/LunaHook/assets/101191390/102eee9b-9cad-4a5e-b8be-66d2322070ca)

**Comment by HIllya51** [comment]
However, I just have found the trial version of the game you mentioned at last issue, you can try it later.

**Comment by HIllya51** [comment]
now you can re download the latest version and try it on your game.

**Comment by HIllya51** [comment]
tested on 仁義なき乙女 Trial version ok.
夢をもう一度＋幸福の風 18禁 dont' have a Trial version, so I can't test. If you test it not work, please tell me.

**Comment by HIllya51** [comment]
all games of 美蕾 which have a Trial version have been tested ok. 
if nothing unexpected happened, 夢をもう一度＋幸福の風 will also work fine.

**Comment by churroloca** [comment]
Seriously, thank you so much! You've made my entire weekend and week. After
reinstalling LunaHook, it showed the engine as GSX and it actually pulled
the code!

Thank you, thank you, thank you. 🥺

**Comment by churroloca** [comment]
Hello,
I'm so sorry to bother you. I reread your email about 夢をもう一度＋幸福の風 and how
it didn't have a trial version. I just tested it and it didn't work with
the LunaHook but it did register the GSX engine.

I can see if there are any trial versions via FANZA game website or DLsite.
If there still isn't any, I could buy it for you - this upcoming week. But
I also understand if I'm asking for far too much.

Looking back on my first ticket, I realised how rude it was of me - to have
asked you to look at three games. I'm so sorry for how I came across. M

I can't thank you enough, for helping me with Jingi Otome. I was resorting
to using DeepL's OCR.
